root/static/scripts/relationship-editor/common/dialog.js

```diff
@@ -431,21 +431,31 @@ class Dialog {
     });
   }
 
+  isUntypedSeries() {
+    return this.source.entityType === 'series' &&
+           this.source.typeID() === '\xA0';
+  }
+
   linkTypeOptions(entityTypes) {
     var options = MB.forms.linkTypeOptions(
       {children: linkedEntities.link_type_tree[entityTypes]},
       this.backward(),
     );
 
     if (this.source.entityType === 'series') {
-      var itemType =
-        MB.seriesTypesByID[this.source.typeID()].item_entity_type;
+      const itemType = this.isUntypedSeries()
+        ? undefined
+        : MB.seriesTypesByID[this.source.typeID()].item_entity_type;
+
+      const allowedPartType = itemType
+        ? PART_OF_SERIES_LINK_TYPES[itemType]
+        : undefined;
```

Comment:
semantic nitpick, but I think null makes more sense than undefined to explicitly indicate "no value" (since we're defining it) unless you need undefined.  also uses less characters :)

basically I'd expect undefined only where we access a property that may not exist, or where we want to pass an "empty" value to a function that has default arguments (since undefined will make it use the default, but null won't)

```suggestion
      const itemType = this.isUntypedSeries()
        ? null
        : MB.seriesTypesByID[this.source.typeID()].item_entity_type;

      const allowedPartType = itemType
        ? PART_OF_SERIES_LINK_TYPES[itemType]
        : null;
```